chapters/convertToAbbreviatedForm.typ

```diff
@@ -135,49 +101,156 @@ They will be stored as a member of the tweak object and then used during the app
     ],
   ),
   caption: "Elements captured for the second refactoring",
-)
+) <second_refactoring_captured_elements_figure>
 
-#pagebreak()
-=== Prerequisites
+=== Call Site Implications <call_site_implications>
+The refactoring must not change the signature of the target method.
+In regard to this specific refactoring, the order of type parameters must stay the same.
+This is only the case if the auto parameters are in the same order as their original template parameters.
+
+For example the two methods in @call_site_differences result in two different signatures.
+When calling these methods with ```cpp foo<int, float>(1.0, 2)``` only the version on the left would compile,
+as the parameter types would be `int` for `param1` and `float` for `param2`.
+The second version would be the opposite, `int` for `param1` and `float` for `param2`, which then breaks the call as the parameter types don't match.
 
 #figure(
-  table(
```

Comment:
@jeremystucki Ich hett die lischte wod usegno hesch drin lah zum zeige was eh funktion alles ha muss demit mer s refactoring chan ahwende. Aso sache wo is prepare inecho würed. Aber evtl. nur sache wos ha muss z.b. "template definition needs to be in place".

denn bide limitations d sache ine tue z.B. ebe das es mit containers ned gaht.

Reply:
Han alles ufteilt uf sneue unterkapitel "call site implications" und "limitations". Isch alles dinne wo in dere Tabelle gsi isch.

Reply:
ok passt, vilt müsst mer denn dete na das ine tue mit de containers